map/src/main/kotlin/edu/artic/map/MapActivity.kt

```diff
@@ -85,4 +88,15 @@ class MapActivity : BaseActivity() {
         Glide.get(this).setMemoryCategory(MemoryCategory.NORMAL)
     }
 
+    override fun onBackPressed() {
+        if (supportFragmentManager.backStackEntryCount == 0) {
+            if (navController.currentDestination?.id == R.id.mapFragment) {
+                val intent = NavigationConstants.HOME.asDeepLinkIntent()
```

Comment:
Please simplify the three lines here and in the other `onBackPressed` overrides using a function in the `navigation` module. Something like

    fun linkHome(): Intent